sale_order_invoicing_queued/data/job_queue.xml

```diff
@@ -0,0 +1,8 @@
+<odoo noupdate="1">
+  <record id="job_function_sale_order_invoicing_action_done" model="queue.job.function">
+    <field name="model_id" ref="sale.model_sale_order" />
+    <field name="method">create_invoices_job</field>
+    <field name="related_action" eval='{"func_name": "custom_related_action"}' />
```

Comment:
Is this one correct? (I don't know the syntax)

Reply:
That's from the example provided in the README file

Reply:
But I think you should provide a valid action name. If not, I would say to remove it and it should take the default one.

Reply:
I was talking about this line, not the whole file.